ospfd/ospf_vty.c

```diff
@@ -9865,6 +9879,58 @@ DEFPY(ip_ospf_capability_opaque, ip_ospf_capability_opaque_addr_cmd,
 }
 
 
+DEFPY(ip_ospf_prefix_suppression, ip_ospf_prefix_suppression_addr_cmd,
+      "[no] ip ospf prefix-suppression [A.B.C.D]$ip_addr",
+      NO_STR "IP Information\n"
+	     "OSPF interface commands\n"
+	     "Supress OSPF prefix advertisement on this interface\n"
+	     "Address of interface\n")
+{
+	VTY_DECLVAR_CONTEXT(interface, ifp);
+	struct route_node *rn;
+	bool prefix_suppression_change;
+
+	struct ospf_if_params *params;
+	params = IF_DEF_PARAMS(ifp);
+
+	if (ip_addr.s_addr != INADDR_ANY) {
+		params = ospf_get_if_params(ifp, ip_addr);
+		ospf_if_update_params(ifp, ip_addr);
+	}
+
+	prefix_suppression_change = (params->prefix_suppression == (bool)no)
```

Comment:
nit: just `params->prefix_suppression == (bool)no`?

Reply:
For conditional assignment, I think it reads better with the condition in parentheses. Similar to the format if a C language "If" statement.

Reply:
I meant just :):
```c
prefix_suppression_change = (params->prefix_suppression == (bool)no);
```

Reply:
Right - I changed.